src/components/task/task-name-text-field/task-name-text-field.tsx

```diff
@@ -48,41 +73,127 @@ export const TaskNameTextField = (props: TaskNameTextFieldProps) => {
                 )}
 
                 {currentStep === 1 && (
-                    <Box sx={{display: "flex", flexDirection: "column", alignItems: "center", gap: "0.5rem"}}>
-                        <Card variant="outlined">
-                            <CardContent sx={{width: 285, display: "flex", justifyContent: "center"}}>
-                                <Typography variant="subtitle2" gutterBottom>{name}</Typography>
+                    <Box
+                        sx={{
+                            display: "flex",
+                            flexDirection: "column",
+                            alignItems: "center",
+                            gap: "1.5rem",
+                        }}
+                    >
+                        <Card
+                            variant="outlined"
+                            sx={{
+                                border: 'none',
+                                borderBottom: `1px solid lightgray`,
+                                backgroundColor: "transparent"
+                            }}
+                        >
+                            <CardContent
+                                sx={{
+                                    display: "flex",
+                                    justifyContent: "center",
+                                    width: 285,
+                                    padding: "0.5rem",
+                                    '&:last-child': {
+                                        paddingBottom: 0,
+                                    },
+                                }}
+                            >
+                                <Typography variant="subtitle1" gutterBottom>{name}</Typography>
                             </CardContent>
                         </Card>
 
                         <TextField
                             label="Description"
                             multiline
-                            rows={2}
-                            sx={{width: "20rem"}}
+                            // rows={2}
+                            sx={{minWidth: "20rem"}}
```

Comment:
Not sure that this is a goof idea. Try to avoid using so huge min width values. It may not fit small screens.